src/UI/src/shared/services/service-helper.js

```diff
@@ -13,6 +13,6 @@ export function status(response) {
 
 export function parseError(error) {
   if (!(error instanceof Error))
-    return new Promise((resolve, reject) => reject(error.json()))
+   return new result((resolve, reject) => reject(error.json()))
```

Comment:
put that back. The error handling you fixed doesn't use this line because the error is an `instanceof Error` and `result` is not a type so newing it doesn't make sense.